client/state/signup/steps/website-content/schema.ts

```diff
@@ -81,6 +85,10 @@ export interface WebsiteContentCollection {
 
 export const initialState: WebsiteContentCollection = {
 	currentIndex: 0,
-	websiteContent: { pages: [], siteLogoUrl: '' },
+	websiteContent: {
+		pages: [],
+		siteLogoSection: { siteLogoUrl: '' },
+		feedbackSection: { genericFeedback: '' },
```

Comment:
This is added anticipating a future use case where these sections might include additional fields in the future. And also modelling all future sections as objects instead of direct data types.